www/js/intro.js

```diff
@@ -23,6 +23,11 @@ angular.module('emission.intro', ['emission.splash.startprefs',
 .controller('IntroCtrl', function($scope, $state, $window, $ionicSlideBoxDelegate,
     $ionicPopup, $ionicHistory, ionicToast, $timeout, CommHelper, StartPrefs, UpdateCheck, $translate, i18nUtils) {
 
+  $scope.surveyState = { 
+    isValid: true, 
+    schema: {'fname':'Jenna', 'lname':'Ruzekowicz'},
+    result: {} 
```

Comment:
Did you try something like 

```suggestion
    result: {'fname': 'Kyle', 'lname': 'Fitzsimmons'} 
```

to confirm that angular -> angular works?